config/overlays/odh/extras/modelregistry_editor_role.yaml

```diff
@@ -0,0 +1,31 @@
+# permissions for end users to edit modelregistries.
+apiVersion: rbac.authorization.k8s.io/v1
+kind: ClusterRole
+metadata:
+  labels:
+    app.kubernetes.io/name: clusterrole
+    app.kubernetes.io/instance: modelregistry-editor-role
+    app.kubernetes.io/component: rbac
+    app.kubernetes.io/created-by: model-registry-operator
+    app.kubernetes.io/part-of: model-registry-operator
```

Comment:
if manifest will be "applied" by ODH operator, can skip this label, operator will add it on the fly.

Reply:
Is this referring to the `part-of` label?

Reply:
Commented part-of label as it will be set by odh operator.